securedrop/tests/encrypt_test.py

```diff
@@ -0,0 +1,91 @@
+import unittest
```

Comment:
There's a lot of duplicate code here. Why not put the new tests you wrote in `client_test.py`?

Reply:
Done.